### DISCUSSION TITLE
Pan settings not copied across from MU3 files

### DISCUSSION THREAD

**Discussion by dave2020X** [musescore/MuseScore#17345] [General]
I have no idea whether this is the right place to write comment or report bugs in MuseScore4 - this is my first attempt at posting in Github. For MU3 I would have put a comment in the musescore.org forum.

As far as I can see mixer pan settings are not copied over to the equivalent settings in MuseScore 4.

**Comment by ghost** [comment]
If reporting a specific problem, then it's best to post it as an issue from [this page](https://github.com/musescore/MuseScore/issues/new/choose). But you'll be happy to know this particular issue has already been posted: https://github.com/musescore/MuseScore/issues/16620 :)
